### DISCUSSION TITLE
One Key, Two Keys, etc. should be condensed into a single Key Conversion mod

### DISCUSSION THREAD

**Discussion by DarnTwine** [ppy/osu#25252] [Ideas]
The Conversion section in the osu!mania tab is too cluttered from the mods One Key, Two Keys, Three Keys, etc. It would be far more convenient and clean to condense them into a singular Key Conversion mod with the specific keycount selectable in Mod Customisation, like with the Difficulty Adjust mod. It just doesn't make sense to have an entire ten mods occupying the list when they are all provide essentially the same functionality.

**Comment by peppy** [comment]
It would definitely tidy things up, now that we have a way to show the detail in the extended region of the mod icon.

Migrating existing plays might be a bit of a pain, but not out of the question.
